### DISCUSSION TITLE
Missing Tooltips for Path Alias Priority Options

### DISCUSSION THREAD

**Discussion by KaseyFarron** [opentoonz/opentoonz#6429] [Ideas]
**Describe the bug**
Tooltips are missing for path alias priority options, specifically for options 1 and 3. Only option 2 has a tooltip, otherwise I can't find out what these options actually do, are 1 and 3 the same? 🤔 

**To Reproduce**
1. Open Preferences > General
2. Open Path Alias Priority drop down box
3. Hover over options 1 or 3

**Expected behavior**
For there to be detailed descriptions of what the options do when hovering.

**Screenshots**
![image](https://user-images.githubusercontent.com/55353161/105509658-a31f9200-5cc5-11eb-9a86-91055dc52d4f.png)

**Device Information:**
 - OpenToonz Version: [1.4 Release]

**Comment by RodneyBaker** [comment]
This should be easy enough to add...although it will need the translations added as well.
Here is where those particular tooltips are defined in the source:

https://github.com/opentoonz/opentoonz/blob/2467105e4ec0a2a3545efe8d33485c8d78d19e43/toonz/sources/toonz/preferencespopup.cpp#L1410

Are there any suggestions for what the Tooltips should display?

Most tooltips are not very lengthy so that is likely why these were left out.
